engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/BinaryDispatchTest.java

```diff
@@ -291,10 +291,14 @@ public void thatArgumentChallengedByManyValues() throws Exception {
     }
   }
 
-  private static void assertContains(String expected, String actual) {
-    if (actual.contains(expected)) {
+  static void assertContains(String expected, String actual) {
+    assertContains("Expecting", expected, actual);
+  }
```

Comment:
FYI: You could conveniently just use `containsString` from hamcrest: `assertThat(someValue, containsString("some text"));` and remove this custom `assertContains` method.